src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -398,7 +398,8 @@ private boolean moveRubricCol(int qnNumber, int colNumber, boolean isMoveLeft) {
         WebElement moveColButton = browser.driver.findElement(By.id(elemId));
 
         if (moveColButton.getAttribute("disabled") == null) {
-            moveColButton.click();
+            waitForElementToBeClickable(moveColButton);
```

Comment:
I think this can be removed. I commented it out and the tests still passed :smile: . Moreover, if the buttons isn't disabled, then I think it should be clickable.